src/app/component/NotSupportedPayment/NotSupportedPayment.style.scss

```diff
@@ -0,0 +1,3 @@
+.NotSupportedPayment {
+    margin-top: 2rem;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4666755)